### PR TITLE
[ORC] add max/min support for orc decimal reader

### PR DESCRIPTION
**Pull request by zuochunwei**
fix the crash caused by `decimal::max/min` In Meituan's orc format environment

### CODE REVIEW

File: velox/dwio/dwrf/reader/SelectiveLongDecimalColumnReader.h [hunk 1]
```diff
@@ -115,7 +115,36 @@ class SelectiveLongDecimalColumnReader
  private:
   template <bool dense>
   void processValueHook(RowSet rows, ValueHook* hook) {
-    VELOX_FAIL("TODO: orc long decimal process ValueHook");
+    printf("orc long decimal process ValueHook: %s\n", hook->dump().c_str());
```
Comment (rui-mo) [inline velox/dwio/dwrf/reader/SelectiveLongDecimalColumnReader.h:118]: Is this a debug message?

File: velox/dwio/dwrf/reader/SelectiveShortDecimalColumnReader.h [hunk 2]
```diff
@@ -121,7 +121,36 @@ class SelectiveShortDecimalColumnReader
  private:
   template <bool dense>
   void processValueHook(RowSet rows, ValueHook* hook) {
-    VELOX_FAIL("TODO: orc short decimal process ValueHook");
+    printf("orc short decimal process ValueHook: %s\n", hook->dump().c_str());
```
Comment (rui-mo) [inline velox/dwio/dwrf/reader/SelectiveShortDecimalColumnReader.h:124]: Same.

File: velox/dwio/dwrf/reader/SelectiveLongDecimalColumnReader.h (hunk 1 above)
Comment (zuochunwei) [reply velox/dwio/dwrf/reader/SelectiveLongDecimalColumnReader.h:118]: yes, I will remove these statements

File: velox/dwio/dwrf/reader/SelectiveShortDecimalColumnReader.h (hunk 2 above)
Comment (zuochunwei) [reply velox/dwio/dwrf/reader/SelectiveShortDecimalColumnReader.h]: OK

**Comment by rui-mo** [comment]
Can we just open a Gluten PR to test this branch?

**Comment by zuochunwei** [comment]
> Can we just open a Gluten PR to test this branch?

OK

**Pull request merged**